src/Message/ItemDataTrait.php

```diff
@@ -23,7 +23,7 @@ public function getItemData(ItemBag $items)
                 'type' => $item->getType(),
                 'name' => $item->getName(),
                 'quantity' => $item->getQuantity(),
-                'tax_rate' => (int) ($item->getTaxRate() * 100),
+                'tax_rate' => \bcmul((string) $item->getTaxRate(),  100),
```

Comment:
`\bcmul` returns a string, while the API expects an integer: https://developers.klarna.com/api/#checkout-api__order__order_lines__tax_rate